app/models/bookings/placement_date.rb

```diff
@@ -39,6 +39,12 @@ class PlacementDate < ApplicationRecord
       presence: true,
       on: :create
 
+    # users manually selecting this only happens when schools are both primary
+    # and secondary, otherwise it's automatically set in the controller
+    validates :supports_subjects,
+      inclusion: [true, false],
+      if: -> { bookings_school&.has_primary_and_secondary_phases? }
```

Comment:
I don't think we need this if either!

Reply:
ah, thinking about it the `on: :create` was because now we'll have old invalid records in the db

Reply:
Having had another look, the reason that ` if: -> { bookings_school&.has_primary_and_secondary_phases? }` is required is so that when dates are created without the user picking a date we're now waiting until we're inside the `if ... valid?` condition before assigning the `supports_subjects` value